third_party/mkl_dnn/mkldnn_v1.BUILD

```diff
@@ -59,9 +70,8 @@ cc_library(
         "src/cpu/**/*.cpp",
         "src/cpu/**/*.hpp",
         "src/cpu/xbyak/*.h",
-    ]) + if_mkl_v1_open_source_only([
-        ":dnnl_config_h",
-    ]) + [":dnnl_version_h"],
+    ]) + [":dnnl_config_h"]
+       + [":dnnl_version_h"],
```

Comment:
One last nit please: Can we merge this into one list? (May need to re-apply formatter.)
```suggestion
    ]) + [":dnnl_config_h", ":dnnl_version_h"],
```

Reply:
Sure . Made the change.